### PR TITLE
Od/tc 00.002.16

### PR DESCRIPTION
**Pull request by Olga-QA-USA**
TC_00.002.16 | New Item > Create Pipeline Project #631

### CODE REVIEW

**Comment by Olga-QA-USA** [comment]
создала ERR_004 |TC_16.002.02 | Dashboard > Create View | Sort items in the view для этой ошибки
Screenshot 2024-12-05 at 22 41 01

**Review by antonbloha — COMMENTED** [review]
test should be written using POM approach.

**Review by Sandarella — APPROVED** [review]

**Review by adiuzhikov — CHANGES_REQUESTED** [review]
Please review the two previous reviews and adhere to the following guidelines:
1. We do not remove `.skip` from old tests. If you wish to write tests in the old format, you are welcome to do so, but you are not required to remove `.skip`.
2. We do not format code written by others.
3. The PR title should be the same as the TC card name.

**Review by NShakiro — APPROVED** [review]

File: cypress/e2e/dashboardCreateViewPOM.cy.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const freestyleProjectPage = new FreestyleProjectPage();
 const header = new Header();
 const myViewsPage = new MyViewsPage();
 
-describe("US_16.002 | Dashboard > Create View", () => {
+describe.skip("US_16.002 | Dashboard > Create View", () => {
```
[the remark this answers is not in the document]
Comment (Olga-QA-USA) [reply cypress/e2e/dashboardCreateViewPOM.cy.js]: It works locally in my computer also, I delete skip already.

**Review by adiuzhikov — APPROVED** [review]

**Pull request merged**